tests/Fixtures/TestBundle/Entity/AlwaysIdentifierDummy.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Core\Annotation\ApiProperty;
+use ApiPlatform\Core\Annotation\ApiResource;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Annotation\Groups;
+
+/**
+ * AlwaysIdentifierDummy.
+ *
+ * @author Alexandre Delplace <alexandre.delplacemille@gmail.com>
+ *
+ * @ApiResource(attributes={
+ *     "normalization_context"={"groups"={"alwaysIdentifierGroup"}, "enable_always_identifier"=true}
```

Comment:
Why do we need this `enable_always_identifier` flag? IMO we can remove it, if the `alwaysIdentifier` is set on a property, we use it, that's all, it doesn't make sense to allow to disable this feature. If you don't want to use it, just don't set the flag.

Reply:
I took example from the max_depth implementation. I'll remove it